### PR TITLE
[DOCS-7617] Update AGS Community files

### PR DESCRIPTION
**Pull request by aritrihore**

### CODE REVIEW

File: governance-services/community/install/zip.md
Comment (anxumalo) [inline governance-services/community/install/zip.md:1]: In this page, let’s change all the **AMP** file versions to be more generic: `23.2.x.xxx`

File: governance-services/community/develop/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ You can download the API Explorer from:
 
 * [Alfresco Nexus repository](https://nexus.alfresco.com/nexus/#welcome){:target="_blank"}
 
-The Governance Services distribution zip contains the `alfresco-governance-services-community-rest-api-explorer-23.1.0.255.war` file.
+The Governance Services distribution zip contains the `alfresco-governance-services-community-rest-api-explorer-23.2.1.xxx.war` file.
```
Comment (anxumalo) [inline governance-services/community/develop/index.md:15]: Let’s change the **WAR** file version to be more generic: `23.2.x.xxx`

```suggestion
The Governance Services distribution zip contains the `alfresco-governance-services-community-rest-api-explorer-23.2.x.xxx.war` file.
```

File: governance-services/community/install/zip.md [hunk 2]
```diff
@@ -23,19 +23,19 @@ The Governance Services Community Edition distribution zip contains the followin
...
 5. Run the following commands to install the AMP files:
 
     ```bash
-    java -jar bin\alfresco-mmt.jar install amps\alfresco-governance-services-community-repo-23.1.0.255.amp tomcat\webapps\alfresco.war
+    java -jar bin\alfresco-mmt.jar install amps\alfresco-governance-services-community-repo-23.2.1.xxx.amp tomcat\webapps\alfresco.war
     ```
 
     ```bash
-    java -jar bin\alfresco-mmt.jar install amps_share\alfresco-governance-services-community-share-23.1.0.269.amp tomcat\webapps\share.war
+    java -jar bin\alfresco-mmt.jar install amps_share\alfresco-governance-services-community-share-23.2.1.xxx.amp tomcat\webapps\share.war
```
Comment (anxumalo) [inline governance-services/community/install/zip.md:38]: Add a hint about how customers should specify the exact released versions - the additional sentence should be at the same indent level as the `bash` commands:

```suggestion
    java -jar bin\alfresco-mmt.jar install amps_share\alfresco-governance-services-community-share-23.2.x.xxx.amp tomcat\webapps\share.war
```

    Replace `x.xxx` with the exact versions provided in the Governance Services Community Edition distribution zip.

**Review by anxumalo — CHANGES_REQUESTED** [review]
I discovered a few minor fixes after checking the released artefacts (see JIRA ticket for additional comments).

**Comment by aritrihore** [comment]
> I discovered a few minor fixes after checking the released artefacts (see JIRA ticket for additional comments).

**Review by anxumalo — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**